UPGRADING.md

```diff
@@ -2,6 +2,41 @@
 
 This document describes breaking changes and how to upgrade. For a complete list of changes including minor and patch releases, please refer to the [changelog][changelog].
 
+## 5.0.0 (unreleased)
+
+Support of keys & values other than strings and Buffers has been dropped. Internally `level-js` now stores keys & values as binary which solves a number of compatibility issues ([Level/memdown#186](https://github.com/Level/memdown/issues/186)). If you pass in a key or value that isn't a string or Buffer, it will be irreversibly stringified.
+
+Existing IndexedDB databases created with `level-js@4` can be read only if they used binary keys and string or binary values. Other types will come out stringified, and string keys will sort incorrectly. Use the included `upgrade()` utility to convert stored data to binary (in so far the environment supports it):
+
+```js
+var leveljs = require('level-js')
+var db = leveljs('my-db')
+
+db.open(function (err) {
+  if (err) throw err
+
+  db.upgrade(function (err) {
+    if (err) throw err
+  })
+})
+```
```

Comment:
Should we include this? I wrote it to see if it could work, and it does for common cases, but not if:

- You target old browsers that don't support binary keys (IE, Edge and a few others, [the ones in red here](https://www.caniuse.com/#feat=indexeddb2))
- You used custom key or value types (numbers etc); these come out stringified
- You mixed key types (e.g. strings and buffers); keys may conflict.

We could (also) choose to drop browsers that don't support binary keys.

Reply:
Yes, lets include this.